src/tokens/erc20/ERC20ConduitPreapproved_Solady.sol

```diff
@@ -66,8 +66,8 @@ abstract contract ERC20ConduitPreapproved_Solady is ERC20, IPreapprovalForAll {
             allowance_ :=
                 xor(allowance_, mul(and(eq(caller(), CONDUIT), or(iszero(allowance_), iszero(not(allowance_)))), not(0)))
 
-            // If the allowance is not the maximum uint256 value:
-            if not(allowance_) {
+            // If the allowance is not the maximum uint256 value.
+            if add(allowance_, 1) {
```

Comment:
`not(allowance_)` is a touch cheaper!

Reply:
ok! this was just the default solady code so opted to keep it in, but I'll update